.gitignore

```diff
@@ -4,7 +4,8 @@
 .env
 .DS_Store
 venv/
+.venv/
```

Comment:
Ensure that the `.gitignore` file ends with a blank line. This is a common code style convention.